src/main/kotlin/blackjack/domain/CardDeck.kt

```diff
@@ -0,0 +1,13 @@
+package blackjack.domain
+
+data class CardDeck(val deck: MutableList<Card>) {
+    init {
+        val defaultDeck = defaultDeckGenerator()
+        require(deck.containsAll(defaultDeck)) { "블랙잭 게임을 하기 위해서는 모든 카드가 준비되어야합니다" }
+    }
+
+    fun drawCard(): Card {
+        require(deck.isNotEmpty()) { "더이상 카드가 남아있지 않습니다" }
+        return deck.removeAt(deck.lastIndex)
```

Comment:
`removeLast()`를 사용하도록 변경해볼 수 있을 것 같아요. 😃
```suggestion
        return deck.removeLast()
```